File: backend/services/post-ts-koa-mongo-fdm/src/common/api/invalid-url-post-content.ts
import { ErrorBody } from "../utils/error";

export const empty: ErrorBody = {
    error: {
        error: "URL_POST_CONTENT_EMPTY",
        reason: "URL post content cannot be empty",
        message: "URL post content cannot be empty",
    }
};

export const tooLong: ErrorBody = {
    error: {
        error: "URL_POST_CONTENT_TOO_LONG",
        reason: "url is too long",
        message: "url is too long",
    }
};

export const invalid: ErrorBody = {
    error: {
        error: "URL_POST_CONTENT_INVALID",
        reason: "URL is invalid",
        message: "URL is invalid",
    }
};

File: backend/services/post-python-fastapi-mongo-fdm/src/api/invalid_url_post_content.py
from typing import Final
from utils.error import ErrorBody

empty: Final[ErrorBody] = {
    "error": {
        "error": "URL_POST_CONTENT_EMPTY",
        "reason": "URL post content cannot be empty",
        "message": "URL post content cannot be empty",
    }
}

tooLong: Final[ErrorBody] = {
    "error": {
        "error": "URL_POST_CONTENT_TOO_LONG",
        "reason": "URL is too long",
        "message": "URL is too long",
    }
}

invalid: Final[ErrorBody] = {
    "error": {
        "error": "URL_POST_CONTENT_INVALID",
        "reason": "URL is invalid",
        "message": "URL is invalid",
    }
}
